src/svgcontext.ts

```diff
@@ -573,6 +573,14 @@ export class SVGContext implements RenderContext {
     return this;
   }
 
+  fillPath(path: string, scale: number, x: number, y: number): this {
+    this.path = path;
+    this.pen.x = NaN;
```

Comment:
FYI: I think we can actually remove the `pen` member as it's not used any more and `CanvasRenderingContext2D` doesn't expose the pen position as far as I know.

Reply:
Yup, it looks like it is no longer used. I wonder when we stopped using it? I will remove it from this PR.